src/main/java/com/hyperwallet/clientsdk/model/HyperwalletPayment.java

```diff
@@ -13,7 +13,12 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public class HyperwalletPayment extends HyperwalletBaseMonitor {
 
+    public static enum Status {CREATED, SCHEDULED, PENDING_ACCOUNT_ACTIVATION, PENDING_TAX_VERIFICATION,
+        PENDING_TRANSFER_METHOD_ACTION, PENDING_TRANSACTION_VERIFICATION, IN_PROGRESS, COMPLETED,
+        CANCELLED, FAILED, RECALLED, RETURNED, EXPIRED}
+
     private String token;
+    private Status status;
```

Comment:
let's simply create the status field as String. The downside of enumeration is that if server-side supports a new Stutes we would need to release new SDK version and push all clients to migrate otherwise it would brek their intgration.

Reply:
@whersbach-hw Since other classes were using enum I followed the same standard. Now I have made changes from enum to String.

Reply:
your previous approach wasn't wrong. Just asked you to change because we had recently issue with those enumerations. Thanks